python_modules/dagster/dagster/cli/__init__.py

```diff
@@ -37,4 +37,4 @@ def group():
 
 
 def main():
-    cli(obj={})  # pylint:disable=E1123
+    cli(auto_envvar_prefix="DAGSTER")  # pylint:disable=E1123
```

Comment:
i worry about `DAGSTER_` colliding with other env vars we use through out the systems since we generally prefix with `DAGSTER_` . Maybe `DAGSTER_CLI_` here?

Reply:
Good idea! Fixed!